File: src/main/java/com/atividade/plp/questao4/Funcionario.java
package com.atividade.plp.questao4;

public class Funcionario {

    String nome;
    double salarioBruto;
    double imposto;

    public Funcionario() {
        //Construtor vazio

    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public double getSalarioBruto() {
        return salarioBruto;
    }

    public void setSalarioBruto(double salarioBruto) {
        this.salarioBruto = salarioBruto;
    }

    public double getImposto() {
        return imposto;
    }

    public void setImposto(double imposto) {
        this.imposto = imposto;
    }


    double SalarioLiquido() {
        double salarioLiquido = this.salarioBruto - this.imposto;
        return salarioLiquido;
    }

    double aumentarSalario(double porcentagem) {
        double salarioLiquido = this.salarioBruto - this.imposto;
        return (salarioBruto * porcentagem) / 100 + salarioLiquido;
    }


}

File: C#/Questao4/Funcionario.cs
using System;

namespace Questao4
{
    public class Funcionario
    {
        String nome;
        double salarioBruto;
        double imposto;

        public string NOME
        {
            get { return nome; }
            set { nome = value; }
        }

        public double SALARIO_BRUTO
        {
            get { return salarioBruto; }
            set { salarioBruto = value; }
        }

        public double IMPOSTO
        {
            get { return imposto; }
            set { imposto = value; }
        }

        public double SalarioLiquido()
        {
            double salarioLiquido = this.salarioBruto - this.imposto;
            return salarioLiquido;
        }

        public double aumentarSalario(double PORCENTAGEM)
        {
            double salarioLiquido = this.salarioBruto - this.imposto;
            return (salarioBruto * PORCENTAGEM) / 100 + salarioLiquido;
        }


    }
}